Makefile

```
@@ -19,3 +19,18 @@ demo:
 	test -d demo/venv || virtualenv demo/venv
 	demo/venv/bin/pip install -e demo
 	DJANGO_SETTINGS_MODULE=demoproject.settings demo/venv/bin/python demo/demoproject/manage.py runserver
+
+.PHONY: demo
+demo-with-fixtures:
+	test -d demo/venv || virtualenv demo/venv
+	demo/venv/bin/pip install -e demo
+	DJANGO_SETTINGS_MODULE=demoproject.settings demo/venv/bin/python demo/demoproject/manage.py makemigrations demoproject
+	DJANGO_SETTINGS_MODULE=demoproject.settings demo/venv/bin/python demo/demoproject/manage.py migrate
+	DJANGO_SETTINGS_MODULE=demoproject.settings demo/venv/bin/python demo/demoproject/manage.py loaddata demo/demoproject/fixtures/meter-readings.json
+	DJANGO_SETTINGS_MODULE=demoproject.settings demo/venv/bin/python demo/demoproject/manage.py runserver
+
+.PHONY: demo
+demo-test:
+	test -d demo/venv || virtualenv demo/venv
+	demo/venv/bin/pip install -e demo
```

This is how you can fix your import hack.

```suggestion
	demo/venv/bin/pip install -e . demo
```